### ISSUE TITLE
Support for proportional fonts

### ISSUE THREAD

**Issue by nickmccurdy** [orta/vscode-twoslash-queries#12]
I exclusively use proportional fonts when coding. I like the idea of this extension, but I wouldn't be able to use it easily because the `^` may not be aligned with the type on my font. Perhaps there could be an alternate syntax that doesn't rely on a monospace font.

**Comment by orta** [comment]
I'm somewhat open to it, but I'm wary the further this extension strays from the twoslash syntax in the TS site / compiler codebase the harder it is to just "get"

Perpahts there could be an identifier lookup in some new syntax similar to how we do types?

```
abc.de
// ^?["de"]
```

Where the ^? implies the line, and then `["de"]` means look through that line until you find a matching identifier and show that

**Comment by nickmccurdy** [comment]
We could also use ESLint style `// next` or `/* enable */` comments.

**Comment by orta** [comment]
No, not in this extension - the key is sticking with the twoslash format and they'll never support those ( but it'd be trivial to fork this extension and add it though, it's only ~200 LOC)
